doc/api/errors.md

```diff
@@ -689,7 +696,13 @@ Used when an attempt is made to send an unsupported "handle" over an IPC
 communication channel to a child process. See [`child.send()`] and
 [`process.send()`] for more information.
 
+<a id="ERR_INVALID_HTTP_TOKEN"></a>
+### ERR_INVALID_HTTP_TOKEN
+
+Used when `options.method` received an invalid http token.
+
 <a id="ERR_INVALID_IP_ADDRESS"></a>
+### ERR_INVALID_IP_ADDRESS
```

Comment:
It seems that I missing some titles for `errors.md` in #14212 . So I added the missing titles in this PR